smoke-tests/playwright.config.js

```diff
@@ -34,10 +34,14 @@ module.exports = defineConfig({
 
   /* Configure projects for major browsers */
   projects: [
-    
+    {
+      name: 'setup',
+      testMatch: /global\.setup\.ts/,
```

Comment:
If I understand it well, this will be always run before the rest, right?

When reading `simple.spec.js`, this is not obvious as it is not directly called. Depending on the framework, I would prefer to use a test or suite setup hook, or a call to a function lazily setting up the session if needed. This allows the reader to find out quickly the prerequisite of each test.

I do not know very well playwright and the Typescript/Javascript ecosystem (and idioms). So, I'm certainly missing something. Could you explain me why you would favor the present approach over the ones I described? (You might want to do that later too ^_^)

Reply:
According to my understanding, when importing `testutils.ts`, you can use `workspacePage` that guarantees you to have logged in. So at least the reader can find out about the need to log in. Still, it's might not clear that you need an existing workspace (maybe a comment would be enough).

Btw, wouldn't it be more clear to import explicitly `workspacePage` so the reader can know where to find it?

Reply:
Multiple runners are started when you launch the tests. Ensuring that only one workspace is created if needed is done as a dependency in the configuration in Playwright. (At least, that's their proposed solution). A different solution could be implemented (checking in the workspacePage fixture and assess there what to do but you might enter in a race condition and have 2+ workspaces created)

Reply:
Regarding the second comment, I used the recommended way to provide a pre-configured page (`workspacePage`).

Reply:
thanks. Indeed, racing condition is an issue that I didn't think about. Thanks. :+1: 

FYI, something I do in such case is to create a unique fixture for each test to make sure they can run independently.